x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/index.tsx

```diff
@@ -342,7 +342,7 @@ export const CreatePackagePolicySinglePage: CreatePackagePolicyParams = ({
     );
   }
 
-  const { agentlessPolicy, handleSetupTechnologyChange, selectedSetupTechnology } =
+  const { isAgentlessEnabled, handleSetupTechnologyChange, selectedSetupTechnology } =
```

Comment:
I know it's a convenience to return `isAgentlessEnabled` in the `useSetupTechnology` as it's being used there, but semantically wouldn't it make more sense to retrieve it from the `useAgentless` hook only? as a new member on the project it is not straightforward to me why this is retrieved from `useSetupTechnology`

```js
const { isAgentlessEnabled } = useAgentless();
const { handleSetupTechnologyChange, selectedSetupTechnology } = useSetupTechnology()
```

Reply:
Yes,  that makes sense.